src/core/common/Breadcrumbs.js

```diff
@@ -10,21 +11,51 @@ const styles = {
         alignItems: "center",
         border: "1px solid #e5e5e5",
         paddingLeft: "10px",
+        backgroundColor: "white",
+    },
+    separator: {
+        width: "0px",
```

Comment:
you can use an integer when you're setting a pixels size in JSS: `width: 0,` - it's faster to write

Reply:
Changed in all places